tutorials/experimental/rcanvas/rcanvas_mt.cxx

```diff
@@ -1,123 +0,0 @@
-/// \file
-/// \ingroup tutorial_rcanvas
-///
-/// This macro demonstrate usage of ROOT7 graphics from many threads
-/// Three different canvases in three different threads are started and regularly updated.
-/// Extra thread created in background and used to run http protocol, in/out websocket communications and process http
-/// requests
-/// Main application thread (CLING interactive session) remains fully functional
-///
-/// \macro_code
-///
-/// \date 2018-08-16
-/// \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
-/// is welcome!
-/// \author Sergey Linev <s.linev@gsi.de>
-
-/*************************************************************************
- * Copyright (C) 1995-2021, Rene Brun and Fons Rademakers.               *
- * All rights reserved.                                                  *
- *                                                                       *
- * For the licensing terms see $ROOTSYS/LICENSE.                         *
- * For the list of contributors see $ROOTSYS/README/CREDITS.             *
- *************************************************************************/
-
-#include "ROOT/RWebWindowsManager.hxx"
-#include "ROOT/RHistDrawable.hxx"
-#include "ROOT/RCanvas.hxx"
-
-#include "TRandom3.h"
-#include "TEnv.h"
-#include "TROOT.h"
-
-#include <thread>
-#include <iostream>
-
-using namespace ROOT::Experimental;
-
-void draw_canvas(const std::string &title, RColor col)
```

Comment:
@linev This tutorials seems to be about using `RCanvas` from multiple threads.  Rather than being removed, should be update to use the old `TH1D` class?

Reply:
Yes, one can create such tutorial with `TH1` class. But one can remove old one as first step.